lib/components/DraftailEditor.test.js

```diff
@@ -31,14 +31,32 @@ describe("DraftailEditor", () => {
     expect(mount(<DraftailEditor />).instance().editorRef).toBeDefined();
   });
 
-  it("#readOnly", () => {
-    expect(
-      shallowNoLifecycle(<DraftailEditor />)
-        .setState({
-          readOnly: true,
-        })
-        .hasClass("Draftail-Editor--readonly"),
-    ).toBeTruthy();
+  describe("readOnly", () => {
+    it("#readOnly from state", () => {
+      expect(
+        shallowNoLifecycle(<DraftailEditor />)
+          .setState({
+            readOnlyState: true,
+          })
+          .hasClass("Draftail-Editor--readonly"),
+      ).toBeTruthy();
+    });
+
+    it("#readOnly from prop", () => {
+      expect(
+        shallowNoLifecycle(<DraftailEditor readOnly />).hasClass(
+          "Draftail-Editor--readonly",
+        ),
+      ).toBeTruthy();
+    });
+
+    it("not #readOnly from prop", () => {
+      expect(
+        shallowNoLifecycle(<DraftailEditor readOnly={false} />).hasClass(
+          "Draftail-Editor--readonly",
+        ),
+      ).toBeFalsy();
```

Comment:
I don’t think this test case is needed since `false` is the default value used in the snapshot tests at the top of the file. I’ll add one for "prop & state".